client/my-sites/checkout/composite-checkout/components/checkout-next-steps.tsx

```diff
@@ -0,0 +1,129 @@
+import { Gridicon } from '@automattic/components';
+import { ResponseCart } from '@automattic/shopping-cart';
+import styled from '@emotion/styled';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { ReactChild, useMemo } from 'react';
```

Comment:
Minor: if you split up type imports and JS imports, the TS compiler can strip out the type imports, which has a slight improvement in file size.

```suggestion
import styled from '@emotion/styled';
import { useTranslate } from 'i18n-calypso';
import { useMemo } from 'react';
import type { TranslateResult } from 'i18n-calypso';
import type { ResponseCart } from '@automattic/shopping-cart';
import type { ReactChild } from 'react';
```

Reply:
Thanks! I have made this change. TIL.